src/main/kotlin/moody/blackjack/domain/user/state/Finished.kt

```diff
@@ -0,0 +1,19 @@
+package moody.blackjack.domain.user.state
+
+import moody.blackjack.domain.card.Cards
+import moody.blackjack.domain.card.Deck
+
+abstract class Finished(cards: Cards) : Started(cards) {
+
+    override fun draw(deck: Deck): UserState {
+        throw IllegalStateException("종료된 상태에서는 카드를 더 받을 수 없습니다.");
+    }
```

Comment:
```suggestion
    override fun draw(deck: Deck): UserState {
        throw IllegalStateException("종료된 상태에서는 카드를 더 받을 수 없습니다.")
    }
```
세미콜론이 있군요! ktlint 체크로 다른 콜론도 지워주면 좋을 것 같아요! 👍

Reply:
ktlint 마지막에 한다 해놓고 까먹고 홀랑 올렸네요 수정하겠습니다!